engine/movie/oak_speech/oak_speech.asm

```diff
@@ -24,12 +29,15 @@ PrepareOakSpeech:
 	call z, InitOptions
 	; These debug names are used for StartNewGameDebug.
 	; TestBattle uses the debug names from DebugMenu.
+	; A variant of this process is performed in PrepareTitleScreen.
 	ld hl, DebugNewGamePlayerName
 	ld de, wPlayerName
 	ld bc, NAME_LENGTH
 	call CopyData
 	ld hl, DebugNewGameRivalName
 	ld de, wRivalName
+	; The two instructions below are the ones in CopyDebugName.
+	; However, attempting to call it here will cause the game to crash.
```

Comment:
What? So... does this routine crash? Does `CopyDebugName` crash?

Reply:
> Does `CopyDebugName` crash?

Yes, I made a quick test to see what happens. Please disregard this stray comment and thank you for pointing it out.